Airwallex/Card/Sources/Card/Internal/AWXDccResponse.m

```diff
@@ -37,4 +37,9 @@ + (id)decodeFromJSON:(NSDictionary *)json {
     return response;
 }
 
++ (instancetype)decodeFromJSONData:(NSData *)data {
```

Comment:
Maybe we can have a parent class that implements AWXCodable so that these classes that inherit the class can have a default implementation of this method, otherwise we need to write same implementation everywhere. (Annoyingly OC doesn't support writing extension method for protocol)

Reply:
We _could_ but my thinking is it may not really be worth it since it'll be the first thing replaced when we rewrite the SDK. I tried to avoid changing the structure of it too much as part of this PR.